### PR TITLE
PromoSanTokenBanner added

### PR DESCRIPTION
**Pull request by peratik**
https://www.notion.so/santiment/Promo-San-token-banner-on-the-Pricing-page-86cb14bedff84f518f54950efd4447a7

- `PromoSanTokenBanner` component added
- `underplans` slot added to easily set without chaning san-webkit repo

image

image

### CODE REVIEW

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  let className = ''
+  export { className as class }
+</script>
+
+<div class="santokenbanner {className} row v-center justify">
+  <div class="leftinfo">
+    <h3 class="h3 mrg--b mrg-l">Hold SAN, Pay Less</h3>
+    <p class="body-2 mrg--b mrg-xl">
+      Enjoy a 20% discount on all Santiment products, and you can burn or provide SAN liquidity for
+      full PRO access.<br />
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:11]: What is the point of the `<br />`

Comment (peratik) [reply src/ui/Pricing/Page/PromoSanTokenBanner.svelte]: Next a should be in the next line.

File: src/ui/Pricing/Page/sanTokenBanner.svg [hunk 2]
```diff
@@ -0,0 +1,88 @@
+  <svg
+    width="240"
+    height="183"
+    viewBox="0 0 240 183"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M41.7763 113.559C17.7239 99.3771 0.988617 44.5 25.9392 17C17.2623 103.465 106.891 72 118.909 123C123.211 141.255 114.587 162.356 97.042 170.719C61.9158 187.46 13.8722 152.896 10.002 105.394C22.5736 115.412 29.429 116.922 41.7763 113.559Z"
+      fill="var(--orange-hover, #FD9317)"
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/sanTokenBanner.svg:10]: CSS variables are not needed

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      class="btn-1 body-2">Buy SAN token</a
+    >
+  </div>
+  <img src={sanTokenBanner} alt="San Token Banner" />
+</section>
+
+<style lang="scss">
+  :global(#plans) {
+    --plans-margin-bottom: 24px;
+  }
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:33]: Change the margin in the component file `src/ui/Pricing/Page/Plans.svelte`

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<script>
+  import sanTokenBanner from './sanTokenBanner.svg'
+  let className = ''
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:3]: Missing new line

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  export { className as class }
+</script>
+
+<section class="santokenbanner row v-center justify {className}">
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:7]: 1. `santokenbanner` is redundant
2. Add `body-2` and remove it from children

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+<style lang="scss">
+  :global(#plans) {
+    --plans-margin-bottom: 24px;
+  }
+
+  .santokenbanner {
+    max-width: 800px;
+    background: var(--whale);
+    border: 1px solid var(--porcelain);
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:38]: Use `border` class

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+<section class="santokenbanner row v-center justify {className}">
+  <div class="left">
+    <h3 class="h3 mrg--b mrg-l">Hold SAN, Pay Less</h3>
+    <p class="body-2 mrg--b mrg-xl">
+      Enjoy a 20% discount on all Santiment products, and you can burn or provide SAN liquidity for
+      full PRO access.
+      <a
+        href="https://academy.santiment.net/san-tokens/san-tokens-holding-benefits"
+        class="btn-0 block"
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:15]: `row` instead of `block`

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    background: var(--whale);
+    border: 1px solid var(--porcelain);
+    border-radius: 8px;
+    padding: 32px 48px;
+    margin-bottom: 80px;
+    margin: 0 auto 80px;
+  }
+
+  .left {
+    width: 424px;
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:46]: Should be responsive

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+    p {
+      color: var(--fiord);
+    }
+
+    a.btn-1 {
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:52]: Just `.btn-1`

File: src/ui/Pricing/Page/PromoSanTokenBanner.svelte [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+      padding: 8px 20px;
+    }
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/PromoSanTokenBanner.svelte:54]: Please, read again 3rd point of the last review

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

**Review by DmitriVanGuard — APPROVED** [review]

**Pull request merged**